src/app/modules/storage-management/repository.model.ts

```diff
@@ -70,7 +70,7 @@ export class Repository {
     public id: string = null;
     public basedir: string = null;
     public policy: RepositoryPolicyEnum = RepositoryPolicyEnum.RELEASE;
-    public implementation = 'file-system';
+    public storageProvider = 'file-system';
```

Comment:
Since you are refactoring with breaking changes - maybe this should also make this field to use `local`, `s3`, `gcs`, `hdfs`, etc? Simply "file-system" is way too vague.

Reply:
Yeah, I agree. `local-file-system`?